### ISSUE TITLE
Salesforce CLI - sfdx force:org:list shows undefined

### ISSUE THREAD

**Issue by SarathKL** [forcedotcom/cli#293]
### Summary
I have 1 Devhub , 2 NonScratch org, 1 scratch org connected.
Type sfdx force:org:list  then Enter. 
Result shows ---

**=== Orgs**
**Columns**   
undefined 
undefined 
undefined 

**Columns**   
undefined 

### Steps To Reproduce:

1.  Type sfdx force:org:list  then Enter. 

### Expected result

Shows all the connected Orgs.

_Describe what should have happened_.

CLI config might have deleted or incorrect.

### Actual result

_Describe what actually happened instead_.

### Additional information

_Feel free to attach a screenshot_.

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
sfdx-cli/7.44.0 darwin-x64 node-v11.0.0
**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-diff 0.0.5
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.4.1 (core)
generator 1.1.2 (core)
mo-dx-plugin 0.2.7
salesforcedx 47.17.0
├─ salesforcedx-templates 47.18.0
└─ salesforce-alm 47.15.0
**OS and version**:

Mac OS - catalina
<img width="561" alt="Screen Shot 2020-02-02 at 8 01 14 PM" src="https://user-images.githubusercontent.com/55328317/73624750-e1282f80-45f6-11ea-8591-b6236f6589df.png">

**Comment by clairebianchi** [comment]
@SarathKL would you be able to provide us with more details on how you authenticated into these orgs, what your project set up is like, and the CLI installation setup? If you uninstall the CLI and start from scratch does that fix the issue? We have not seen this before and it is working on our end.

**Comment by SarathKL** [comment]
Thanks @clairebianchi ! uninstalling and re-installling the CLI fixed the issue.

**Comment by smukov** [comment]
I'm facing the exactly same issue with both `sfdx force:org:list` and `sfdx force:package:list`. A workaround for me is to append `--json` to the commands - I haven't tried reinstalling the CLI yet. 

```
$ sfdx --version
sfdx-cli/7.47.0-0e5d18732f darwin-x64 node-v10.15.3

$ sfdx plugins --core
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.2.3 (core)
@oclif/plugin-help 2.2.3 (core)
@oclif/plugin-not-found 1.2.3 (core)
@oclif/plugin-plugins 1.7.9 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-trust 3.0.7 (core)
analytics 1.7.1 (core)
generator 1.1.2 (core)
mo-dx-plugin 0.2.8
salesforcedx 48.2.0
├─ @salesforce/sfdx-plugin-lwc-test 0.1.5
├─ salesforcedx-templates 48.3.0
└─ salesforce-alm 48.3.0
```

Is there an alternative way to fix this without reinstalling?
